CHANGELOG.md

```diff
@@ -1,3 +1,21 @@
+# dbt_intercom v1.2.0
+
+## Schema/Data Change
+**1 total change • 0 possible breaking changes**
+
+| Data Model(s) | Change type | Old | New | Notes |
+| ------------- | ----------- | ----| --- | ----- |
+| All models | New column | | `source_relation` | Identifies the source connection when using multiple Intercom connections |
+| `stg_intercom__tag` | New Data Type | `name` categorized as timestamp | `name` categorized as string | Modifies name to its proper string format. |
```

Comment:
```suggestion
| `stg_intercom__tag` | New data type | `name` categorized as timestamp | `name` categorized as string | Modifies name to its proper string format. |
```
Just to align with the casing from the previous row (alternatively can adjust line 8 to be New Column)